index.html

```diff
@@ -812,31 +727,32 @@ <h3>
           Release wake lock algorithm
         </h3>
         <p>
-          To <dfn>release a wake lock</dfn> for a given {{WakeLock}} object
-          |lock:WakeLock|, run these steps <a>in parallel</a>:
+          To <dfn>release a wake lock</dfn> for a given |lockPromise:Promise| and
+          |type:WakeLockType|, run these steps <a>in parallel</a>:
         </p>
         <ol class="algorithm" data-link-for="WakeLock">
-          <li>Let |type:WakeLockType| be |lock|'s <a>type</a>.
+          <li>Let |document:Document| be the <a>responsible document</a> of the
+          <a>current settings object</a>.
           </li>
-
           <li>Let |record| be the <a>platform wake lock</a>'s <a>state
-          record</a> associated with |type|.
+            record</a> associated with |document| and |type|.
           </li>
-          <li>If the promise |lock|.{{[[requestPromise]]}} not `undefined` and
-            is pending, reject it with an "{{AbortError}}" {{DOMException}} and
-            <a>abort the request steps</a> for the associated running
-            <a>request()</a> algorithm.
+          <li>If |record|.{{[[ActiveLocks]]}} does not contain |lockPromise|,
+            abort these steps.
```

Comment:
How could this happen? Can you leave a note explaining this state?

Reply:
Added note